Tests/RxSwift/ToggleBoolTests.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ToggleBoolTests.swift
+//  RxSwiftExt
+//
+//  Created by Vincent on 20/02/2018.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import XCTest
+
+import RxSwift
+import RxTest
+import RxSwiftExt
+
+class ToggleBoolTests: XCTestCase {
+    
+    func testStartWithFalse() {
+        let originalSequence = Observable.of(1, 2, 3, 4)
+        let correct: [Recorded<Event<Bool>>] = [
+            next(0, false),
```

Comment:
I think I would prefer testing timed events instead of all events being false, so we can confirm the time of the emissions are respected. You can use createColdObservable in RxTest to do that.

Reply:
Done